src/backend/expungeservice/models/helpers/record_summarizer.py

```diff
@@ -0,0 +1,65 @@
+from expungeservice.models.expungement_result import ChargeEligibilityStatus
+from expungeservice.models.record_summary import RecordSummary
+from typing import Dict
+
+
+class RecordSummarizer:
+    @staticmethod
+    def summarize(record):
```

Comment:
We should probably start a habit of adding types to public methods in classes `def summarize(record) -> RecordSummary:`. This allows you to jump to the return value class definition on most IDEs.